### PR TITLE
Document creating backend pages

### PR DESCRIPTION
**Pull request by I-Valchev**

### CODE REVIEW

File: docs/extensions/pages.md [hunk 1]
```diff
@@ -1,4 +1,140 @@
 Adding Pages in the backend
 ===========================
 
-[todo] Write some stuff here adding pages in the backend, that kinda look like they fit in with the overall backend.
+## Create a page in the Bolt's backend
+
+You can add pages to Bolt's backend by creating a custom
+Controller in the `src` folder of your project.
```
Comment (nestordedios) [inline docs/extensions/pages.md:7]: Hi @I-Valchev , Nice Docs!! 👏 

Maybe the word **Controller** should be capitalized or not capitalized in the entire document to keep it consistent?

File: docs/extensions/pages.md [hunk 2]
```diff
@@ -1,4 +1,140 @@
...
+
+To configure it as a backend controller, two steps are necessary:
+* The controller class needs to implement the `BackendZoneInterface`
```
Comment (nestordedios) [inline docs/extensions/pages.md]: See here the word **controller** not being capitalized

File: docs/extensions/pages.md [hunk 3]
```diff
@@ -1,4 +1,140 @@
...
+class UserContentController extends TwigAwareController implements BackendZoneInterface
+{
+    /**
+     * @Route("/content-user/", name="app_content_user")
+     */
+    public function viewEdits(): Response
+    {
+        // content_user.html.twig is a custom file 
+        // that needs to be located in the `templates`
+        // folder in the root of your project.
```
Comment (nestordedios) [inline docs/extensions/pages.md:35]: Maybe that could be wrapped in a block comment?
```
    /**
     * content_user.html.twig is a custom file
     * that needs to be located in the `templates`
     * folder in the root of your project.
     */
```

File: docs/extensions/pages.md [hunk 4]
```diff
@@ -1,4 +1,140 @@
...
+            'title' => 'User content'
+        ]);
+    }
+}
+```
+
+
+### Defining the controller route with the backend url prefix
+
+Localte the `config/routes.yaml` file and add the following **after**
```
Comment (nestordedios) [inline docs/extensions/pages.md]: **Localte** looks like a typo here

**Review by nestordedios — CHANGES_REQUESTED** [review]
Nice work @I-Valchev !! 👍  I pointed out some things that you may want to reconsider.

File: docs/extensions/pages.md (hunk 1 above)
Comment (I-Valchev) [reply docs/extensions/pages.md:7]: Yes, good idea! I've capitalised them.

File: docs/extensions/pages.md (hunk 3 above)
Comment (I-Valchev) [reply docs/extensions/pages.md:35]: The `/**` annotation is used for PhpDoc (e.g. to document functions, classes). In that case, this is more like a comment for the person copy-pasting the example.

File: docs/extensions/pages.md (hunk 4 above)
Comment (I-Valchev) [reply docs/extensions/pages.md]: Yes, good catch. I type like a monkey 🙈

File: docs/extensions/pages.md (hunk 1 above)
Comment (nestordedios) [reply docs/extensions/pages.md:7]: Perfect! 👍

File: docs/extensions/pages.md (hunk 3 above)
Comment (nestordedios) [reply docs/extensions/pages.md:35]: Ah ok. I always though that block comments where also meant to comment out paragraphs of text. In that case, I haven't said anything 🙈

**Review by nestordedios — APPROVED** [review]
It looks great! 👍 This is ready for consumption.

**Comment by bobdenotter** [comment]
Part of #1168

**Pull request merged**